### PR TITLE
feat: virtuse driver

### PR DESCRIPTION
**Pull request by sajcics**

### CODE REVIEW

File: drivers/virtuse.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  const tickers = throttleMap(
+    Object.keys(pairs),
+    async (pair) => {
+      try {
+        const data = await request({
+          url: 'https://virtuse.exchange/api/v1',
+          method: 'post',
+          form: {
+            method: 'public/get_ticker',
+            pair,
```
Comment (nickpater) [inline drivers/virtuse.js]: Hi Stella! I just checked the documentation of Virtuse and saw that if you omit the pair, then you get a full list of all the pairs/tickers. If we would use that one, this driver would simplify a lot!

Comment (sajcics) [reply drivers/virtuse.js]: @nickpater can you please send me url of what exact url you mean of ? I am not sure that I can find that on their documentation.

Comment (nickpater) [reply drivers/virtuse.js]: Certainly! Although it is the same URL, but then without sending a specific pair. Something like this:
```Javascript
const data = await request({
  url: 'https://virtuse.exchange/api/v1',
  method: 'post',
  form: {
    method: 'public/get_ticker',
  },
  headers: {
    'Content-Type': 'application/json',
  },
});
```

Comment (sajcics) [reply drivers/virtuse.js]: You are right. I don't know how I miss that. Here is new update.

**Pull request merged**